OpenRA.Mods.Common/Traits/Captures.cs

```diff
@@ -23,6 +23,9 @@ public class CapturesInfo : ITraitInfo
 		[Desc("Types of actors that it can capture, as long as the type also exists in the Capturable Type: trait.")]
 		public readonly HashSet<string> CaptureTypes = new HashSet<string> { "building" };
 
+		[Desc("Triggers TransformOnCapture tagged by these types")]
+		public readonly HashSet<string> TransformTypes = new HashSet<string> { "husk" };
```

Comment:
I gave a thought on it but I think it best suits there since to transform or not to transform depends on the capturer.